src/main/webapp/js/ideas/showIdea.directive.js

```diff
@@ -0,0 +1,29 @@
+(function() {
+	'use strict';
+
+	angular
+		.module('app.directives')
+		.directive('showIdea', showIdea);
+
+	function showIdea() {
+		return {
+			restrict: 'E',
+			replace: true,
+			scope: {
+				idea: '@',
+				photo: '@',				
+				title: '@',
+				description: '@',
+				created: '@',
+				author: '@',			
+				tag: '@',
+				rtng: '@',
+				ngModel: '=',
```

Comment:
May be, it makes sense to make required methods belong to directive: move required methods from ideaController to directive's link function (if they are not required in other places except directive)